findbugs-exclude-filter.xml

```diff
@@ -133,4 +133,11 @@
             <Bug pattern="IMPROPER_UNICODE" />
         </Match>
     </FindBugsFilter>
+    <FindBugsFilter>
+        <Match>
+            <Class name="org.wso2.carbon.identity.rest.api.server.notification.template.v1.core.TemplateTypeService"/>
+            <Method name="resetTemplateType"/>
+            <Bug pattern="IMPROPER_UNICODE"/>
```

Comment:
Shall we add a comment explaining why this exclusion has been added?

(The comment for the already existing exclusion for the same "IMPROPER_UNICODE" bug mentions the usage of the "toUpperCase" method, but the "resetTemplateType" method mentioned in the new filter here does not use the "toUpperCase" method, so it seems the cause is different.)

Reply:
The original comment mentions that the false positive was `while using "toUpperCase"`, but this exclusion has been added for the `resetTemplateType` method (according to [line 139](https://github.com/wso2/identity-api-server/pull/629/files#diff-ba8062618586e91a337270936da281ac569eb8d1e261209312ed9d463f0c8091R139)), which has no usages of `toUpperCase`. 

Presumably, the false positive in the new case was due to the `equalsIgnoreCase` method. It would be better if we can update the comment to mention this.

Reply:
addressed with https://github.com/wso2/identity-api-server/pull/629/commits/d4df259f15fdad63db051f2ed29160d53e091aec